superchain/configs/mainnet/superlumio.yaml

```diff
@@ -16,3 +16,5 @@ genesis:
     hash: "0x01711dc53a82967a34f4b38f7dff3ae8fa760882bb1e2ace50bb01e9ff420a9b"
     number: 0
   l2_time: 1708984631
+
+ecotone_time: 1708984632
```

Comment:
This is almost certainly incorrect -- I have reached out to try and get the correct value. It "works" in the sense that the genesis block hash validation check passes, and the GasPriceOracle validation check knows to use the "ecotone" path.

Reply:
Fixed with updated times